lib/health_checking/health_checks/health_check.rb

```diff
@@ -0,0 +1,51 @@
+module HealthChecks
+  module HealthCheck
+
+    def self.included(base)
+      base.extend ClassMethods
```

Comment:
I've been trying to get better at metaprogramming to :smiley: 

I'm still learning what's best practice, I found an example in the sidekiq library that uses a similar technique

https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/worker.rb